maven-build/action.yml

```diff
@@ -0,0 +1,29 @@
+name: Maven build
+description: build a maven jar
+runs:
+  using: "composite"
+  steps:
+    - id: get-build-parameters
+      run: |
+        branch_name=$(echo ${{ github.ref }} | cut -d "/" -f 3)
+        revision=$branch_name-$(git log -n1 --date=format-local:%G%m%d-%H%M --format='%ad' | tr -d " ")
```

Comment:
Do you know if this has to be a date-time string? I wonder if it can be `GITHUB_RUN_NUMBER`

> A unique number for each run of a particular workflow in a repository. This number begins at 1 for the workflow's first run, and increments with each new run. This number does not change if you re-run the workflow run.

https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

Reply:
Good question, I have no idea, I just copied what was done in the Jenkins global pipeline. I did not put much thoughts on that, here is what the generated jar look like `setup-store-11-20210401-1055-d22e44c37.jar`. I am in favour of keeping things as they where in Jenkins and leave it to them to modify if they want.